src/bluetooth/DataAccess.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2021
+ */
+
+/**
+ * Helper methods to perform data access functionality for breach calculations
+ */
+import VACCINE_ENTITIES from '../utilities/modules/vaccines/constants';
+
+export class DataAccess {
```

Comment:
I wonder if we should have a name which is somewhat related to cold chain stuff `ColdChainDataAccess` 😕 ..? 🤷

Reply:
Hm good point, DataAccess is a bit vague 😆 , is it more appropriate to prepend these things with `Vaccine` or `ColdChain`? I guess it's not always going to be vaccines? I created some constants below using `Vaccine` but am now reconsidering renaming those too 🤔 

I also kind of struggled with whether this should go in the Bluetooth folder or the `utilities/modules/vaccines` folder, lol.

Reply:
Yeah I'm not sure. I generally search files by name so I usually don't care so much for this type of thing.. I think either vaccines or coldchain is suitable in our context/names we use for stuff

Reply:
👍  went with Vaccine for now ab82519d